sendemail.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import os
+
+KEY = os.getenv('MAILGUN_API_KEY')
+SANDBOX = os.getenv('MAILGUN_DOMAIN')
+ENVIRONMENT = os.getenv('ENVIRONMENT')
+
+
+def send_email(subject, body):
+
+    if ENVIRONMENT == 'dev':
+        print('Status: 200')
```

Comment:
Questa print e' un po' inutile o sbaglio? Non c'e' mai stata una chiamata http in questo caso, e quindi non c'e' mai stato uno status di ritorno.
Questa print serve per feedback utente, quindi facciamo qualcosa come:
```python
print('** Email **')
print(body)
```